sources/platform/console/store.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Store
+description: Explore Apify's Store, browse and select Actors, search by criteria, sort by relevance, and adjust settings for immediate or future runs.
+sidebar_position: 2
+category: platform
+slug: /console/store
+---
+
+**Explore Apify's Store, browse and select Actors, search by criteria, sort by relevance, and adjust settings for immediate or future runs.**
+
+---
+
+![apify-console-store](./images/console-store.png)
+
+The Store is a place where you can explore a variety of Actors, both created and maintained by Apify or our community members. To enhance your browsing experience, you can employ several strategies.
+Utilize the search box at the top of the page to find Actors by service names, such as TikTok, Google, Facebook, or by their authors.
+Alternatively, explore Actors grouped under predefined categories accessible via the side menu.
+You can also organize the results from the store by different criteria, including:
+
+* relevance
+* popularity
+* newest
+* recent updates
+
+upon selecting an Actor from the store, you'll be directed to its specific page. Here, you can configure the settings for your future Actor run, save these configurations for later use, or run the Actor immediately.
```

Comment:
```suggestion
Upon selecting an Actor from the store, you'll be directed to its specific page. Here, you can configure the settings for your future Actor run, save these configurations for later use, or run the Actor immediately.
```

Reply:
No, but it's not really a continuation; you've started a new sentence. And it having a sentence continue from a numbered list doesn't make for easy reading.